com.getferdi.Ferdi.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was it submitted upstream?

Reply:
I tried to automatically create the <version> tag but i did not spend that much time on it. For now i will just maintain the metainfo externally. I talked about it with the maintainers in the issue linked above. As soon as this https://github.com/electron-userland/electron-builder/issues/1993 is build into the electron builder I can just add it from there.